### PR TITLE
Added 'type' type to CreateStaticStorage & CreateSQLiteStorage

### PR DESCRIPTION
**Pull request by NextThread**

### CODE REVIEW

File: packages/workspace/src/blob/local-static-storage.ts [hunk 1]
```diff
@@ -1,34 +1,13 @@
...
-  'fb0SNPtMpQlzBQ90_PB7vCu34WpiSUJbNKocFkL2vIo=',
-  'k07JiWnb-S7qgd9gDQNgqo-LYMe03RX8fR0TXQ-SpG4=',
-  'i39ZQ24NlUfWI0MhkbtvHTzGnWMVdr-aC2aOjvHPVg4=',
-  '0hjYqQd8SvwHT2gPds7qFw8W6qIEGVbZvG45uzoYjUU=',
-  'qezoK6du9n3PF4dl4aq5r7LeXz_sV3xOVpFzVVgjNsE=',
-  'sNVNYDBzUDN2J9OFVJdLJlryBLzRZBLl-4MTNoPF1tA=',
-  'Bd5F0WRI0fLh8RK1al9PawPVT3jv7VwBrqiiBEtdV-g=',
-  '029uztLz2CzJezK7UUhrbGiWUdZ0J7NVs_qR6RDsvb8=',
-  '5Cfem_137WmzR35ZeIC76oTkq5SQt-eHlZwJiLy0hgU=',
-];
```
Comment (himself65) [inline packages/workspace/src/blob/local-static-storage.ts:20]: Dont remove these lines

**Review by himself65 — CHANGES_REQUESTED** [review]

File: packages/workspace/src/blob/local-static-storage.ts [hunk 2]
```diff
@@ -1,34 +1,31 @@
...
+   'fb0SNPtMpQlzBQ90_PB7vCu34WpiSUJbNKocFkL2vIo=',
+   'k07JiWnb-S7qgd9gDQNgqo-LYMe03RX8fR0TXQ-SpG4=',
+   'i39ZQ24NlUfWI0MhkbtvHTzGnWMVdr-aC2aOjvHPVg4=',
+   '0hjYqQd8SvwHT2gPds7qFw8W6qIEGVbZvG45uzoYjUU=',
+   'qezoK6du9n3PF4dl4aq5r7LeXz_sV3xOVpFzVVgjNsE=',
+   'sNVNYDBzUDN2J9OFVJdLJlryBLzRZBLl-4MTNoPF1tA=',
+   'Bd5F0WRI0fLh8RK1al9PawPVT3jv7VwBrqiiBEtdV-g=',
+   '029uztLz2CzJezK7UUhrbGiWUdZ0J7NVs_qR6RDsvb8=',
+   '5Cfem_137WmzR35ZeIC76oTkq5SQt-eHlZwJiLy0hgU=',
+ ];
```
Comment (himself65) [inline packages/workspace/src/blob/local-static-storage.ts:20]: don't move inside the function, this is unnecessary

File: packages/workspace/src/blob/local-static-storage.ts (hunk 1 above)
Comment (NextThread) [reply packages/workspace/src/blob/local-static-storage.ts:20]: hi, as I'm new here, 
I'm creating a new branch and doing the same and raising a new pr.

syntax mismatched :(

File: packages/workspace/src/blob/local-static-storage.ts [hunk 3]
```diff
@@ -1,34 +1,31 @@
...
   return {
+    type: 'Static',
     crud: {
       get: async (key: string) => {
         if (key.startsWith('/static/')) {
           const response = await fetch(key);
           return response.blob();
-        } else if (predefinedStaticFiles.includes(key)) {
-          const response = await fetch(`/static/${key}.png`);
-          return response.blob();
```
Comment (himself65) [inline packages/workspace/src/blob/local-static-storage.ts:31]: revert the code

**Review by himself65 — CHANGES_REQUESTED** [review]

**Review by himself65 — CHANGES_REQUESTED** [review]
Thanks for your contribution. I think we don't really need `type`

**Comment by NextThread** [comment]
> Thanks for your contribution. I think we don't really need `type`

hey pls, this is my first open source contribution
I was working from yesterday

**Pull request closed without merging**